packages/ember-htmlbars/tests/integration/escape_integration_test.js

```diff
@@ -1,117 +0,0 @@
-import run from 'ember-metal/run_loop';
-import EmberView from 'ember-views/views/view';
-import compile from 'ember-template-compiler/system/compile';
-
-import { set } from 'ember-metal/property_set';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-
-var view;
-
-QUnit.module('ember-htmlbars: Escaped Integration', {
-  teardown() {
-    runDestroy(view);
-
-    view = null;
-  }
-});
-
-QUnit.test('should read from a global-ish simple local path without deprecation', function() {
-  view = EmberView.create({
-    context: { NotGlobal: 'Gwar' },
-    template: compile('{{NotGlobal}}')
-  });
-
-  expectNoDeprecation();
-  runAppend(view);
-
-  equal(view.$().text(), 'Gwar');
-});
-
-QUnit.test('should read a number value', function() {
-  var context = { aNumber: 1 };
-  view = EmberView.create({
-    context: context,
-    template: compile('{{aNumber}}')
-  });
-
-  runAppend(view);
-  equal(view.$().text(), '1');
-
-  run(function() {
-    set(context, 'aNumber', 2);
-  });
-
-  equal(view.$().text(), '2');
-});
-
-QUnit.test('should read an escaped number value', function() {
-  var context = { aNumber: 1 };
-  view = EmberView.create({
-    context: context,
-    template: compile('{{{aNumber}}}')
-  });
-
-  runAppend(view);
-  equal(view.$().text(), '1');
-
-  run(function() {
-    set(context, 'aNumber', 2);
-  });
-
-  equal(view.$().text(), '2');
-});
-
-QUnit.test('should read from an Object.create(null)', function() {
```

Comment:
I think that test is slightly different: the test I added was for `{{nullObject}}` (which apparently does not work in htmlbars :P), whereas this is testing `{{nullObject.foo}}` works. So it seems fine to keep both!